internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -1024,30 +1025,57 @@ func (r *AgentServiceConfigReconciler) newAssistedServiceDeployment(ctx context.
 	return deployment, mutateFn, nil
 }
 
+func (r *AgentServiceConfigReconciler) getMustGatherImages(log logrus.FieldLogger, instance *aiv1beta1.AgentServiceConfig) string {
```

Comment:
Would it be possible to add some docstrings on what this function does, the expected input, and the expected output?

Reply:
Sure. I will add this and the extra documentation in the following PR.